server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -987,67 +928,71 @@ private long balanceTablets() {
       final Map<DataLevel,Set<KeyExtent>> partitionedMigrations = partitionMigrations();
       int levelsCompleted = 0;
 
-      for (DataLevel dl : DataLevel.values()) {
-        if (dl == DataLevel.USER && tabletsNotHosted > 0) {
-          log.debug("not balancing user tablets because there are {} unhosted tablets",
-              tabletsNotHosted);
-          continue;
-        }
-
-        if ((dl == DataLevel.METADATA || dl == DataLevel.USER)
-            && !partitionedMigrations.get(DataLevel.ROOT).isEmpty()) {
-          log.debug("Not balancing {} because {} has migrations", dl, DataLevel.ROOT);
-          continue;
-        }
+      try (var tabletsMutator = getContext().getAmple().mutateTablets()) {
```

Comment:
When switching this to conditional mutations, will need to do the writes per data level because ample does not support writing conditional mutations to different data levels at the same time.